### PR TITLE
fix(wizard-modal): added support for RTL

### LINKED ISSUE #452: RTL: WizardModal: Various
- General modal issues - see #444 
  - Title overlaps "x" close button.
  - Too much padding between right edge of button and text 

image
image

- No padding btw step circle and text (as there is in the LTR version)

image

### PR DESCRIPTION
**Pull request by enricoberti**
Closes #452

**Summary**

- the wizard modal (title, content, spacing, buttons) was broken on RTL

**Change List (commits, features, bugs, etc)**

- moved away from styled component to pure css

**Acceptance Test (how to verify the PR)**

image

### CODE REVIEW

File: src/components/WizardModal/_wizard-modal.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
+@import '../../globals/vars';
+
+.#{$iot-prefix}--wizard-modal {
+  .bx--modal-container {
```
Comment (tay1orjones) [inline src/components/WizardModal/_wizard-modal.scss]: These will need to use the configurable prefix from Carbon. I think it's just `prefix`?
```suggestion
  .#{prefix}--modal-container {
```

File: src/components/WizardModal/_wizard-modal.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    min-width: 410px;
+  }
+
+  .bx--progress {
```
Comment (tay1orjones) [inline src/components/WizardModal/_wizard-modal.scss]:
```suggestion
  .#{prefix}--progress {
```

File: src/components/WizardModal/_wizard-modal.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    /* need to pad some space for the focus outlines*/
+    padding: 3px;
+  }
+  .bx--progress-step {
```
Comment (tay1orjones) [inline src/components/WizardModal/_wizard-modal.scss]:
```suggestion
  .#{prefix}--progress-step {
```

File: src/components/WizardModal/_wizard-modal.scss [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    max-width: 150px;
+  }
+  > div + div {
+    padding-top: 1rem;
+  }
+
+  .bx--modal-content {
```
Comment (tay1orjones) [inline src/components/WizardModal/_wizard-modal.scss]:
```suggestion
  .#{prefix}--modal-content {
```

File: src/components/WizardModal/_wizard-modal.scss [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    padding-top: 1rem;
+    padding-left: 1rem;
+    padding-bottom: 7rem;
+  }
+}
+
+html[dir='rtl'] {
+  .#{$iot-prefix}--wizard-modal {
+    .bx--modal-header,
+    .bx--modal-content {
```
Comment (tay1orjones) [inline src/components/WizardModal/_wizard-modal.scss]:
```suggestion
    .#{prefix}--modal-header,
    .#{prefix}--modal-content {
```

File: src/components/WizardModal/_wizard-modal.scss [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+      padding-left: 20%;
+      padding-right: 3rem;
+    }
+    .bx--progress-step svg {
```
Comment (tay1orjones) [inline src/components/WizardModal/_wizard-modal.scss]:
```suggestion
    .#{prefix}--progress-step svg {
```

**Comment by enricoberti** [comment]
thanks @tay1orjones! regarding the spacing, i actually moved 1:1 from the styled component before, but 👍 for moving to vars where possible!

**Pull request merged**